external/fv3fit/fv3fit/sklearn/_min_max_novelty_detector.py

```diff
@@ -113,6 +110,9 @@ def predict(self, data: xr.Dataset) -> xr.Dataset:
         score_dataset = stacked_scores.to_dataset(name=self._SCORE_OUTPUT_VAR).unstack(
             SAMPLE_DIM_NAME
         )
+        score_dataset[self._CENTERED_SCORE_OUTPUT_VAR] = score_dataset[
```

Comment:
Maybe a bit simpler to also return a tuple here and avoid the `_CENTERED_SCORE_OUTPUT_VAR`. This will also allow mypy to check that the data is being passed correctly between routines which will make this code more maintainable.

Reply:
Would that break the type signature of the `Predictor` class's `predict` function? It seems to call for a single dataset having each element of `output_variables`. Is your proposal for a pair of data arrays, one for the regular scores and one for the centered score?

Reply:
Oh...i'm not proposing changing the Predictor API. I didn't see that the `NoveltyDetector` ABC is a subclass of Predictor too. I was confused since `OutOfSampleModel` is also a `Predictor`.

Anyway, feel free to ignore this comment...just feels like some layers/inheritance/methods could be removed as you suggest [here](https://github.com/ai2cm/fv3net/pull/1945#discussion_r927208280)...either in this or a future PR.